magellan-library/src/main/java/com/wealthfront/magellan/core/Step.kt

```diff
@@ -8,21 +9,28 @@ import android.view.View
 import androidx.annotation.VisibleForTesting
 import androidx.viewbinding.ViewBinding
 import com.wealthfront.magellan.lifecycle.LifecycleAwareComponent
-import com.wealthfront.magellan.view.lifecycleWithContext
+import com.wealthfront.magellan.lifecycle.lifecycleWithContext
 
 abstract class Step<V : ViewBinding>(
   createBinding: (LayoutInflater) -> V
 ) : Navigable, LifecycleAwareComponent() {
 
   private var viewState: SparseArray<Parcelable>? = null
+  final override var activity: Activity? = null
+    private set
 
   var viewBinding: V? by lifecycleWithContext { createBinding.invoke(LayoutInflater.from(it)) }
     @VisibleForTesting set
 
   final override var view: View? by lifecycleWithContext { viewBinding!!.root }
     @VisibleForTesting set
 
+  override fun onCreate(context: Context) {
```

Comment:
This will get overridden by inheritors of this class. We should probably create an activity LifecycleComponent instead for this reference

Reply:
Oops. Missed that. good catch!